src/pages/patterns/fluid-styles/index.mdx

```diff
@@ -0,0 +1,289 @@
+resizes.
+
+<Row>
+<Column colLg={8}>
+
+![Example of fluid](/images/fluid-pattern-overview.png)
+
+</Column>
+</Row>
+
+<Caption>
+  A fluid text input and fluid buttons inside a fluid modal container using the
+  condensed grid mode.
+</Caption>
+
+### Key terms
+
+| Term              | Definition                                                                                                                                                                                                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Fixed width       | The width of an element is set to a specific value and does not change as the browser resizes.                                                                                                                                                                                |
+| Fluid width       | The width of an element is responsive, often proportionally changing between breakpoints or is set as a percentage of a container.                                                                                                                                            |
+| Default style     | The primary style and behavior of a component that may respond either in a fluid or fixed-width manner depending on layout and responsive strategy. Default styling is more productive in nature and is most commonly used across product UI.                                 |
+| Fluid style       | An alternative style and behavior available for certain components that always has a fluid width. Fluid components are percentage-based as they comprise part of a greater whole. Fluid styling is more expressive and is used to create drama and emphasis in an experience. |
+| Productive moment | Experiences that keep the user focused and able to complete complex tasks. Productive elements may appear more conservative, confident, and efficient on the whole.                                                                                                           |
+| Expressive moment | Experiences that create a desired impact or pause in a layout. Expressive elements are often employed in long-form, editorial experiences, when users are learning and exploring. Expressive elements may evoke more personality and emotion.                                 |
+
+### When to use fluid styles
+
+- Use in expressive or important moments, to better capture the user’s attention
+  and offer a rhythmic break to the productive experience.
+- Use within or attached to compound components.
+- Use in spacious layouts where the fluid variants have space to expand and
+  breath.
```

Comment:
Should this end in `breathe` instead of `breath`?
```suggestion
- Use in spacious layouts where the fluid variants have space to expand and
  breathe.
```